Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -574,6 +574,13 @@ class SearchViewController: SiteTableViewController,
             }
         case .firefoxSuggestions:
             let firefoxSuggestion = firefoxSuggestions[indexPath.row]
+            if let clickInfo = firefoxSuggestion.clickInfo {
+                let position = SearchListSection.allCases.prefix { $0.rawValue < indexPath.section }
+                    .reduce(0) { $0 + self.tableView(tableView, numberOfRowsInSection: $1.rawValue) }
+                + indexPath.row
```

Comment:
I think this should be:

```suggestion
                + indexPath.row + 1
```

(Either here, or in `recordFirefoxSuggestSelectionTelemetry`—but `indexPath`s are zero-based, IIUC, but we want 1-based per the description in `metrics.yaml`).